modules/administration/pages/virtual-hosts.adoc

```diff
@@ -107,8 +107,16 @@ image::systems_virtual_host_managers_vmware.png[scaledwidth=80%]
 
 [IMPORTANT]
 ====
-If you do not refresh the data from a new Virtual Host Manager, host data will not be inventoried and therefore will not be displayed under menu:Systems[Virtual Systems].
+If you do not refresh the data from a new Virtual Host Manager, host data will not be inventoried and therefore will not be displayed under menu:System List[Virtual Systems].
 ====
 +
 
-. View inventoried ESXi host objects and resources by selecting menu:Systems[Virtual Systems] .
+. View inventoried ESXi host objects and resources by selecting menu:System List[Virtual Systems].
+
+[NOTE]
+====
+Depending on your environment and settings, you might receive an ``invalid certificate`` error when you connect to the ESXi server from a web browser using HTTPS.
```

Comment:
```suggestion
Depending on your environment and settings, you might receive an ``invalid certificate`` error when you connect to the ESXi server from a Web browser using HTTPS.
```
According to the style guide, we used to capitalize Web.

Reply:
Seems old fashioned ... Do we capitalise Internet as well?

Reply:
Actually, on second thoughts, we can probably just remove the word 'web' ... which removes the problem ;)